login-fire-common-behavior.html

```diff
@@ -154,6 +154,16 @@
       },
 
       /**
+        * Whether to show the view at sign up or sign in
+        *
+        * @type {Boolean}
+        */
+        showSignUp: {
```

Comment:
It will be used in login-fire-buttons to allow for customizing "Sign in with" or "Sign up with". See https://github.com/convoo/login-fire/blob/46_localization/locales.json#L9

Just an extra tiny change